infrastructure/keymanager/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "keymanager"
+version = "0.0.1"
+
```

Comment:
```suggestion
edition = "2018"
```

Reply:
you have to state edition 2018 to use it

Reply:
Already added, thanx